### PR TITLE
PLAT-13612: Adjust how CLI is tracked in Amplitude

### PR DESCRIPTION
**Pull request by alaisgomes**
## Ticket

PLAT-13612
_Related PRs:_ https://github.com/crowdbotics/vue-dash-5104/pull/4663

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [x] Minor changes

Did you make changes to modules or create a new module?

- [ ] Yes, and have read the [modules updates checklist](https://github.com/crowdbotics/modules/#modules-updates-checklist) documentation and followed the instructions.
- [x] No.

## Changes introduced
Updated amplitude tracking to respect the send the same properties as the Vue dashboard. The following events are being tracked:
- List Modules
- Publish Modules
- View Module Details
- Create Module
- Archive Module
- Unarchive Module
- Upgrade Scaffold

**Notes**: 
- we don't have any app-specific actions in the CLI, so app-related properties were skipped
- Although I introduced the concept of `currentUser` global object, I did not replace other `get /user/` API calls with this global object yet (I wanted to keep the changes scoped to amplitude-related code only)

## Test and review
Test and see if the events mentioned above are being sent to Amplitude.

### CODE REVIEW

File: scripts/amplitude/wrapper.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { configFile } from "../utils/configFile.js";
+import { AMPLITUDE_API_KEY, OPT_IN_NAME } from "./config.js";
+import { init, track, Identify, identify } from "@amplitude/analytics-node";
+import { currentUser } from "../utils/user.js";
+
+class AmplitudeWrapper {
+  get userType() {
+    // TODO: Implement once we have the data available in the UserSerializer
```
Comment (robester0403) [inline scripts/amplitude/wrapper.js:8]: Just doubling checking that there's a ticket for this further down the road.

File: scripts/amplitude/wrapper.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  }
+
+  async loadAndIdentify(user) {
+    await currentUser.setUser(user);
+    if (!currentUser.get("id")) return;
+
+    const identifyEvent = new Identify();
+    identifyEvent.set("Django Id", currentUser.get("id"));
+    identify(identifyEvent, { user_id: currentUser.get("email") });
```
Comment (robester0403) [inline scripts/amplitude/wrapper.js:45]: Not really sure but are we guaranteed to get an email from the user response? otherwise ignore this comment

**Review by robester0403 — DISMISSED** [review]
Looks good with small optional comments

File: scripts/amplitude/wrapper.js (hunk 1 above)
Comment (alaisgomes) [reply scripts/amplitude/wrapper.js:8]: I don't think there's one yet, but this was defined as a mandatory property in the spreadsheet, so I added this to not forget. I will ask Bryan, probably should create the ticket now

File: scripts/amplitude/wrapper.js (hunk 2 above)
Comment (alaisgomes) [reply scripts/amplitude/wrapper.js:45]: Updated in cd145dab

**Review by robester0403 — APPROVED** [review]

**Pull request merged**